packages/next/src/server/config.ts

```diff
@@ -300,6 +324,12 @@ function assignDefaults(
     }
   }
 
+  warnOptionHasBeenDeprecated(
+    result,
+    'experimental.appDir',
+    'App router is enabled by default now, `experimental.appDir` option can be safely removed.',
```

Comment:
small copy suggestion
```suggestion
    'App router is available by default now, `experimental.appDir` option can be safely removed.',
```